### PR TITLE
[website] fixing footer position in safari

### PR DESCRIPTION
**Pull request by jareknowotka**
## Motivation

Documentation website has a broken footer position in Safari v.14 OSX. The fix was tested in all major browsers:
OSX: Safari v14; Firefox v81, v82; Brave v86
Windows10: Edge v86; IE11; Firefox v12; Chrome v86; Opera v72

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

Open website in Safari and verify if the footer is not sticking at the top of the page.

Website before the fix:

Screenshot 2020-10-28 at 09 51 37

Website after the fix:

Screenshot 2020-10-28 at 09 51 01

### CODE REVIEW

**Comment by JoelMarcey** [comment]
@jareknowotka I think this should be good, but waiting for you to sign the CLA before reviewing and possibly approving.

**Comment by jareknowotka** [comment]
Thank you @JoelMarcey, in fact I had trouble signing CLA as individual as well. I keep getting "Mailing Address is required" although I provide my address. Seams like a form validation issue. Unfortunately not a part of this repository, so unable to fix. 

In the meanwhile I went the corporate path, signed the docs and waiting for approval.

**Comment by bmwill** [comment]
@jareknowotka Thanks for the bug report on the CLA process. We have someone taking a look at fixing it.

**Comment by maxwellgordon** [comment]
@jareknowotka CLA bug has been solved. You should be able to finish the Individual CLA signing process now!

**Review by JoelMarcey — APPROVED** [review]
I think this is ok. I am actually not quite sure what the use case would be to have both `height` and `min-height` to be exactly the same. Wouldn't `min-height` override `height` in this case anyway? Maybe my CSS-foo needs some work here.

**Pull request closed without merging**
